update to the latest doc links in readme

cacheung opened the pull request:
update to the latest doc links in readme

<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->

## Related Issue

<!--- This project only accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

## Motivation and Context

<!--- Why is this change required? What problem does it solve? -->

## How Has This Been Tested?

<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):

## Types of changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

cacheung committed:
update to the latest doc links in readme

update to the latest doc links in readme

emdobrin commented on README.md:
```diff
@@ -59,7 +59,7 @@ make run-tests-locally
 ## Usage
 
-### [Audience Manager](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-audience-manager)
+### [Audience Manager](https://developer.adobe.com/client-sdks/previous-versions/documentation/adobe-audience-manager/)
```
Can we remove the link in the header since we already mentioned it before?

cacheung committed:
Update readme link and package json react native version

Update readme link and package json react native version

cacheung committed:
update react native version

update react native version

cacheung committed:
update react native version

update react native version

cacheung committed:
update package json

update package json

cacheung committed:
remove android sample app in ci

remove android sample app in ci

cacheung committed:
update react native version to 0.64.4

update react native version to 0.64.4

cacheung committed:
revert ci script

revert ci script

cacheung committed:
update build.gradle

update build.gradle

cacheung committed:
update build.gradle

update build.gradle

cacheung committed:
update build.gradle

update build.gradle

cacheung committed:
update build.gradle

update build.gradle

cacheung committed:
update build.gradle

update build.gradle

cacheung committed:
remove react native in build.gradle

remove react native in build.gradle

cacheung committed:
update build.grade

update build.grade

cacheung committed:
update react-native version

update react-native version

cacheung committed:
update build gradle

update build gradle

cacheung committed:
revert build gradle change

revert build gradle change

cacheung committed:
Comment out android build in CI

Comment out android build in CI

cacheung committed:
remove extra repositories in build gradle file

remove extra repositories in build gradle file

cacheung committed:
update xcode version for ci

update xcode version for ci

cacheung committed:
revert package json version

revert package json version

cacheung committed:
Update xcode version in ci

Update xcode version in ci

cacheung committed:
Update react version in sample app

Update react version in sample app

cacheung committed:
revert package.json version in sample app

we need to keep the react version to 17.0.1 and react-native to 0.64.x for dependencies issue.

yangyansong-adbe reviewed (approved):
LGTM
